text.go

```diff
@@ -109,17 +109,19 @@ func (t *Text) Nc() int {
 	return t.file.b.Nc()
 }
 
-// whatstring provides an easy-reading version of the Text usage.
-func (t *Text) whatstring() string {
-	switch t.what {
+// String returns a string representation of the TextKind.
+func (tk TextKind) String() string {
```

Comment:
There are a few places (3 I think) where I called `whatstring` that I suspect are commented out. Can you update them? I think they're all in `text.go`. Could you update?